policyengine_us/variables/gov/states/vt/tax/income/credits/ctc/vt_ctc.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class vt_ctc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Vermont child tax credit"
+    definition_period = YEAR
+    unit = USD
+    reference = "https://casetext.com/statute/vermont-statutes/title-32-taxation-and-finance/chapter-151-income-taxes/subchapter-002-taxation-of-individuals-trusts-and-estates/section-5830f-see-note-vermont-child-tax-credit/1"
+    defined_for = StateCode.VT
+
+    def formula(tax_unit, period, parameters):
+        # Get age status of all people in the tax unit.
+        person = tax_unit.members
+        age = person("age", period)
+        p = parameters(period).gov.states["vt"].tax.income.credits.ctc
+        eligible = age < p.age_threshold
+        count_eligible = tax_unit.sum(eligible)
+        # Get maximum credit amount.
+        max_credit = p.amount * count_eligible
+        # Get adjusted gross income.
+        agi = tax_unit("adjusted_gross_income", period)
+        # Reduce credit amount over the phaseout range.
+        excess_agi = max_(agi - p.reduction.start, 0)
+        # using a mask to avoid division by zero.
+        increment = p.reduction.increment
+        increments = np.zeros_like(increment)
+        mask = increment > 0
+        # increments = excess_agi / p.reduction.increment
+        increments[mask] = excess_agi[mask] / increment[mask]
+        percent_reduction = p.reduction.amount * increments
+        # Return reduced amount.
+        return max_(max_credit - percent_reduction, 0)
```

Comment:
use max() instead of max_ might be better?

Reply:
What's the difference between them?

Reply:
I think we can use max_( - this will eliminate vectorized outputs